### PR TITLE
Add earliest_for_each_object to audit event

### PR DESCRIPTION
**Pull request by MuriloDalRi**
Update APIClient according to [#596](https://github.com/alphagov/digitalmarketplace-api/pull/596)

### CODE REVIEW

**Comment by risicle** [comment]
Ah - version bump

**Comment by MuriloDalRi** [comment]
Better now?

**Comment by risicle** [comment]
yup, but we'd better hold off on merging it till the api implementation is merged

File: dmapiclient/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = '8.10.0'
+__version__ = '8.20.0'
```
Comment (samuelhwilliams) [inline dmapiclient/__init__.py]: Should be 8.11.0

Comment (TheDoubleK) [inline dmapiclient/__init__.py]: Nice spot, although we might be up to 20 by the time this all lands :p

**Pull request merged**